charts/team-ns/values.yaml

```diff
@@ -1,4 +1,4 @@
-toolsVersion: 1.3.5
+toolsVersion: v1.4.11
```

Comment:
@Morriz why did I have to put `v1.4.11` here and not `1.4.11` as the old version was just `1.3.5`

Reply:
we are using `v*` tags now?

Reply:
we have to remove this line as we don't use the global `toolsVersion` in any code now

Reply:
True, this is kind of dead configuration, not used in `team-ns` chart